### PR TITLE
Add e2e tests

### LINKED ISSUE #4: Writing unit and e2e tests
Frameworks:

- e2e - puppeteer
- unit - jest

### PR DESCRIPTION
**Pull request by ruddi10**
fixes #4 

Steps to test this PR-
Pull updated backend from go-master
Use make run to run backend.
Pull frontend code.
Run yarn install
Run yarn build
yarn global add serve
serve -s build
Finally run yarn test:e2e

### CODE REVIEW

**Comment by ruddi10** [comment]
@muskankhedia this is the pr for testing

**Comment by ruddi10** [comment]
This PR can be reviewed once backend integration or is merged

File: src/components/RouteList/index.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```
Comment (aquibbaig) [inline src/components/RouteList/index.tsx:1]:
```
import PropTypes from "prop-types";
..
..
..
..
RouteList.propTypes = {
  routes: PropTypes.array.isRequired,
  error: PropTypes.string.isRequired,
};
```

Comment (ruddi10) [reply src/components/RouteList/index.tsx:1]: Here error can be undefined too

**Review by muskankhedia — APPROVED** [review]

**Pull request merged**